spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/firestore/GcpFirestoreProperties.java

```diff
@@ -39,6 +39,11 @@
 
 	private String projectId;
 
+	/**
+	 * The host and port of the Firestore emulator service; can be overridden to specify an emulator
+	 */
+	private String hostPort = "firestore.googleapis.com:443";
```

Comment:
In `GcpDatastoreProperties`, we just call it `host`. Can we stick with that convention please?

Reply:
Actually, maybe it's Datastore that needs to be fixed.

Reply:
Hmm, so should we change this to `host` or leave as `hostPort`? Am fine either way; I guess changing datastore might break some people? :/